templates/build/index.html

```diff
@@ -18,7 +18,9 @@ <h1>Your Ubuntu image is building</h1>
   {% if 'raspi' in build_info["metadata_override"]["subarch"] %}
   <div class="row">
     <div class="col-8">
-      <p>Meanwhile, start getting ready the things you’ll need to flash your device:</p>
+      <p>Meanwhile, start getting ready the things you’ll need to set up your device:</p>
+      <p>If you’re using a Raspberry Pi you can use the official Raspberry Pi Imager to flash your image to your SDcard.</p>
```

Comment:
- pi imager link - https://www.raspberrypi.org/downloads/
- etcher link - https://www.balena.io/etcher/

I think we should add links

Reply:
I added them to the copy doc as well

Reply:
Thanks, I've added the links